packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -540,6 +540,59 @@ class _UiKitViewGestureRecognizer extends OneSequenceGestureRecognizer {
   }
 }
 
+/// The render box for an embedded win32 HWND.
+class RenderWin32View extends RenderBox {
```

Comment:
Why isn't this a `PlatformViewRenderBox` if it's a `RenderBox` intended to implement a platform view?

Reply:
Due to how input and hit tests are handled by the Windows kernel as regards HWNDs, I do not foresee using a `PlatformViewHitTestBehavior` or any `OneSequenceGestureRecognizers` ourselves for this `RenderBox`.